app/assets/stylesheets/components/page_specific/_home_top.scss

```diff
@@ -0,0 +1,193 @@
+// Banner at the top of the home page
+//
+// Styleguide Home Top
+
+.home-top {
+  background: image_url('illustrated_people.png') 0 100% no-repeat;
+  background-size: 135%;
+  margin: 0;
+  position: relative;
+  width: 100%;
+  display: block;
+  padding-bottom: 50%;
+
+  @include respond-to($mq-m) {
+    background-position: 165% 108%;
+    background-size: 70%;
+    padding-bottom: 0;
+  }
+
+  @include respond-to($mq-l) {
+    background-size: 65%;
+    background-position: 141% 100%;
+
+    @if ($responsive == false) {
+      width: $mq-l;
+      margin: 0 auto;
+      float: none;
+    }
+  }
+
+  @include respond-to($mq-xl) {
+    background: none;
+    overflow: hidden;
+  }
+}
+
+.home-top-trust {
+  position: relative;
+
+  @include respond-to($mq-xl) {
+    position: absolute;
+    top: 0;
+    width: 1000px;
+  }
+}
+
+.home-top__image {
+  @include respond-to($mq-xl) {
+    width: 1350px;
+    height: 309px;
+    margin: 0 auto;
+    position: relative;
+    background: image_url('illustrated_people.png') 100% 52% no-repeat;
+    background-size: 49%;
+
+    .theme-cy & {
+      height: 351px;
+      background-position: 100% 100%;
+      background-size: 51%;
+    }
+  }
+}
+
+.home-top-trust__heading {
+  @extend %font-heading-heavy;
+  @include body(36, 42);
+  color: $color-heading-extra-small;
+  padding-top: $baseline-unit*5;
+  padding-bottom: $baseline-unit*5;
+  letter-spacing: -1.5px;
+  margin: 0;
+
+  @include respond-to(px(600), px(719)) {
+    max-width: 500px;
+  }
+
+  @include respond-to($mq-m) {
+    @include body(48, 54);
+
+    .theme-cy & {
+      max-width: 600px;
+    }
+  }
+
+  @include respond-to($mq-l) {
+    max-width: 530px;
+
+    .theme-cy & {
+      max-width: 500px;
+    }
+  }
+
+  @include respond-to($mq-xl) {
+    max-width: 700px;
+
+    .theme-cy & {
+      max-width: 750px;
+    }
+  }
+}
+
+.home-top-trust__content {
+  @extend .unstyled-list;
+  @extend .list-benefits;
+  margin-bottom: $baseline-unit*2;
+
+  > li {
+    @include body(16,24);
+    margin-bottom: $baseline-unit;
+    @extend %font-heading-medium;
+
+    @include respond-to($mq-m) {
+      max-width: 45%;
+    }
+
+    @include respond-to($mq-l) {
+      max-width: none;
+
+      .theme-cy & {
+        max-width: 570px;
+      }
+    }
+  }
+
+  @include respond-to($mq-l, $mq-xl) {
+    .theme-cy & {
+      max-width: 50%;
+    }
+  }
+}
+
+.home-top-promo {
+  clear: both;
+
+  @include respond-to(px(600), px(719)) {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+  }
+
+  @include respond-to($mq-l) {
+    clear: none;
+    position: absolute;
+    bottom: 0;
+    right: 3%;
+    margin: 0;
+  }
+}
+
+.home-top-promo__text {
+  @extend %font-heading-heavy;
+  @include body(22, 30);
+  background: #000;
```

Comment:
Other option is to use $color-outer-space (based on name from that name that colour website) for the existing MAS black and change everywhere that uses it.